db/seed_helper.rb

```diff
@@ -1,16 +1,14 @@
 require 'open-uri'
 require 'json'
-# require 'dotenv'
-# Dotenv.load('.env')
+require 'dotenv'
 
-url = "https://api.edamam.com/api/food-database/v2/parser?app_id=#{ENV.fetch('EDAMAM_APP_ID')}&app_key=#{ENV.fetch('EDAMAM_API_KEY')}&nutrition-type=cooking"
-
-5.times do |index|
+Dotenv.load('.env') # run the script from the repo root: ruby db/seed_helper.rb
+filters= "&diet=balanced&dishType=Bread&dishType=Cereals&dishType=Main%20course&dishType=Preps&dishType=Salad&dishType=Sandwiches&dishType=Side%20dish&dishType=Soup&dishType=Starter&imageSize=THUMBNAIL"
+url = "https://api.edamam.com/api/recipes/v2?type=public&app_id=#{ENV.fetch('EDAMAM_RECIPE_API_APP_ID')}&app_key=#{ENV.fetch('EDAMAM_RECIPE_API_KEY')}#{filters}"
```

Comment:
Are `EDAMAM_RECIPE_API_APP_ID` and `EDAMAM_RECIPE_API_KEY` different than the ones already in Heroku?

Reply:
Yes, they are keys for the [Recipe API](https://developer.edamam.com/edamam-recipe-api), which is different from the[ Food Database API ](https://developer.edamam.com/food-database-api)

Reply:
Thanks for clarifying. I'll see about updating the keys on Heroku.

Reply:
@chrononaut76 the keys have been added to the config var settings on heroku ✅